lib/polarshift/request.rb

```diff
@@ -307,6 +305,22 @@ def update_caseruns(project_id, run_id, updates)
         )
       end
 
+      def update_caseruns_smart(project_id, run_id, updates, timeout: 360)
+        res = nil
+        success = wait_for(timeout, interval: 30) {
+          res = update_caseruns(project_id, run_id, updates)
+          if res[:success]
+            return res
+          else
+            logger.warn %Q{got status "#{res[:exitstatus]}" updating test run "#{run_id}" in project "#{project_id}":\n#{res[:response]}}
+          end
```

Comment:
Last statement needs to be `false` in case a repeat is needed. I'm not sure what `logger.warn` returns but we shouldn't rely on it.

Reply:
@akostadinov  Updated.

Reply:
I'm not sure what changed. I don't see a `false` added anywhere. Did you push your change?